README.md

```diff
@@ -1,14 +1,38 @@
-### Salesforcebulk ESB Connector
+# Salesforce Bulk EI Connector
 
-The Salesforcebulk connector allows you to access the Salesforcebulk REST API through WSO2 ESB. SalesforceBulk is a RESTful API
-that is optimal for loading or deleting large sets of data.You can use it to query, insert, update, upsert, or delete a large number of records
-asynchronously by submitting batches that Salesforce processes in the background.
+The Salesforce Bulk [Connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to access the [Salesforce Bulk REST API](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/asynch_api_intro.htm) through WSO2 Enterprise Integrator (WSO2 EI). SalesforceBulk is a RESTful API that allows you to either quickly load large sets of your organisation’s data into Salesforce or delete large sets of your organisation’s data from Salesforce. You can use SalesforceBulk to query, insert, update, upsert or delete a large number of records asynchronously, by submitting the records in batches. Salesforce can process these batches in the background.
```

Comment:
The Salesforce Bulk Connector allows you to access the Salesforce Bulk REST API through WSO2 Enterprise Integrator (WSO2 EI). SalesforceBulk is a RESTful API that allows you to quickly load large sets of your organization’s data into Salesforce or delete large sets of your organization’s data from Salesforce. You can use SalesforceBulk to query, insert, update, upsert, or delete a large number of records asynchronously, by submitting the records in batches. Salesforce can process these batches in the background.

Reply:
Fixed in a0448a990213d3cdb3f3ae158ef2307753bad900.